src/utils/tests/validation.test.js

```diff
@@ -172,3 +173,110 @@ describe('validation.js - validateVmName', () => {
     expect(validateVmLikeEntityName('vm1', vmSettings, props)).toEqual(getValidationObject(VIRTUAL_MACHINE_EXISTS));
   });
 });
+
+describe('validation.js - isValidMAC', () => {
```

Comment:
we could also consider moving this test to a separate file

Reply:
As with moving the function, I think we should do this only if the function grows in complexity in the future.